Src/bat_charge.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * bat_charge.c
+ *
+ *  Created on: Sep, 26 2020
+ *      Author: Luke Oxley
+ */
+#include "bat_charge.h"
+
+uint8_t temp_array[READ_MSG_SIZE];
+uint8_t read_byte = 0;
+uint8_t write_data[WRITE_MSG_SIZE + 1];
+
+void init_BQ21062(uint8_t * write_data);
+
+void task_manage_charger() 
+{
+    uint8_t currentF, voltageF, tempF;
```

Comment:
make sure to initialize all variables.
also 1 variable decl per line.

also also  don't have the "F" postfix, might give the impression of an int being a floating point number.

also also also camel case is reserved for function calls in our standard, while snake case is for variables
camel case: thisIsAFunction
snake case: this_is_a_variable